src/main/java/frc/robot/commands/Test.java

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.DriveTrain;
+
+public class Test extends CommandBase {
+  /** Creates a new Test. */
+  //Timer timer;
+  DriveTrain driveTrain;
+  double goal;
```

Comment:
@rice31415 , do we care if fields are public or private for commands?

Reply:
in the past we were kinda inconsistent with it and did both default and private but i checked the example command and its supposed to be private with subsystems as final too